File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        
        PriorityQueue<Integer> MaxHeap = new PriorityQueue<>();

        for(int num : nums){

             MaxHeap.add(num);

             if(MaxHeap.size() > k){

                  MaxHeap.poll();
             }
        }
             return MaxHeap.peek();
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        
         priority_queue<int , vector<int>, greater<int>>maxheap;

         for(const auto& num : nums){

             maxheap.push(num);
             if(maxheap.size() > k){

                  maxheap.pop();
             }
         }
            return maxheap.top();
    }
};